.github/workflows/release.yaml

```diff
@@ -44,10 +44,12 @@ jobs:
             ~/go/bin
           key: pre-release-check-${{ runner.os }}-go-${{ matrix.go-version }}-${{ hashFiles('**/go.sum', 'Makefile') }}
 
-      # ubuntu is missing wixl https://github.com/actions/virtual-environments/issues/3857
-      # ubuntu is also missing osslsigncode (no issue, yet)
-      - name: "Install GNOME msitools (wixl) and osslsigncode"
-        run: sudo apt update -qq && sudo apt install -qq -y wixl osslsigncode
+      # windows-2022 is missing osslsigncode (no issue, yet)
+      - name: "Install osslsigncode, infozip; setup wix"
+        run: |
+          choco install osslsigncode -y
+          choco install zip -y
```

Comment:
installing zip instead of 7-zip so this part of the Makefile is left unchanged. This also makes it possible to run on non-Windows although it is currently skipped.